lib/elasticband/query/score_function/score_mode.rb

```diff
@@ -0,0 +1,21 @@
+module Elasticband
+  class Query
+    class ScoreFunction
+      class ScoreMode < ScoreFunction
```

Comment:
See the others examples, there are a lot of them sharing the same behaviour or close to that, because the approach is always focusing on verbosity.